core/src/main/java/com/epam/esm/jpa/OrderJpaRepository.java

```diff
@@ -0,0 +1,28 @@
+package com.epam.esm.jpa;
+
+import com.epam.esm.model.entity.OrderEntity;
+import org.springframework.data.domain.Pageable;
+import org.springframework.data.jpa.repository.Query;
+
+import java.util.List;
+
+public interface OrderJpaRepository extends BaseRepository<OrderEntity, Long> {
+
+    /**
+     * Find orders by user id
+     *
+     * @param userId order id
+     * @return OrderEntity
+     */
+    @Query("select oe from OrderEntity oe where oe.userEntity.id = :userId")
```

Comment:
Doesn't this method work without defining exact query?

Reply:
I used @Query, because I want to search orders by userId, not by UserEntity. Spring Data could not do it for me.